### ISSUE TITLE
Neovide Compatibility Issue with Unextracted nvim.appimage on Kali GNU/Linux

### ISSUE THREAD

**Issue by jerrygems** [neovide/neovide#1975]
### Neovide Compatibility Issue with Unextracted nvim.appimage on Kali GNU/Linux

**If you're using the neovide with an old version of nvim or if you're trying it with nvim.appimage from the official github repo without extracting it on _Kali GNU/Linux Rolling x86_64_ then you may have unexpected behavior in which if you move your cursor it will get closed or will stop working**
**And it also won't work if you try this**
```bash
neovide --neovim-bin neovim.appimage
```

**To Reproduce**
1. Install the neovide from source installation (https://neovide.dev/installation.html#linux-source) or official github repo
2. Install nvim.appimage or nvim via apt pkg manager (in my case apt installs the NVIM v0.7.2)
3. After installation just run neovide it won't show the error there but you can check the logs file or can try creating logs using ```neovide --log```
4. Your logs will be something like as show below you can check by scrolling

**Expected behavior**
it should have worked fine with nvim.appimage without extracting but it doesn't work untill we install it via these steps 
```bash
./nvim.appimage --appimage-extract
./squashfs-root/AppRun --version
sudo mv squashfs-root /
sudo ln -s /squashfs-root/AppRun /usr/bin/nvim
nvim
```

**Desktop :**
 - OS: [e.g. macOS 11.4]
 - Neovide Version [0.11.0]
 - Neovim Version [v0.7.2]

**Logs**
```bash
┌──(spidey㉿Spiderverse)-[~]
└─$ cat neovide_backtraces.log
2023-08-12 10:43:25 - Neovide panicked with the message 'Mouse Input Failed: NeovimError(Some(1), "invalid button or action")'. (File: src/bridge/ui_commands.rs; Line: 69, Column: 18)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: start_thread
             at ./nptl/pthread_create.c:444:8
  15: __GI___clone3
             at ./misc/../sysdeps/unix/sysv/linux/x86_64/clone3.S:81

2023-08-12 10:43:25 - Neovide panicked with the message 'Could not send serial ui command: SendError { .. }'. (File: src/bridge/ui_commands.rs; Line: 259, Column: 22)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: start_thread
             at ./nptl/pthread_create.c:444:8
  15: __GI___clone3
             at ./misc/../sysdeps/unix/sysv/linux/x86_64/clone3.S:81

2023-08-12 10:43:25 - Neovide panicked with the message 'called `Result::unwrap()` on an `Err` value: SendError { .. }'. (File: src/event_aggregator.rs; Line: 51, Column: 28)
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: __libc_start_call_main
             at ./csu/../sysdeps/nptl/libc_start_call_main.h:58:16
  22: __libc_start_main_impl
             at ./csu/../csu/libc-start.c:360:3
  23: <unknown>
```

**If it's happening to you as well then try it out with neovim installation with extracting the appimage that is shown above**

**Comment by fredizzimo** [comment]
We debugged this on discord. Note that running the `nvim.appimage` directly works, so it's not a problem with that.

This might be fixed by https://github.com/neovide/neovide/pull/1946, but I'm not sure.

**Comment by falcucci** [comment]
seems to be fixed by #1946 but got no response since then. @jerrygems reach out in case it persists.
